src/common/home/sections/features.jsx

```diff
@@ -0,0 +1,54 @@
+import { Link } from "react-router-dom";
+import { RiSlideshow4Line } from "react-icons/ri";
+import { BiShareAlt, BiAddToQueue } from "react-icons/bi";
+import { FaLightbulb } from "react-icons/fa";
+import './features.css';
+
+export default function SectionFeatures() {
+  return (
+    <section className="home-features">
+      <ul className="list-home-features">
+        <li className="home-features-item">
```

Comment:
You can also make an array of object and loop through them for avoiding repetitive code and also easier to add features later

Reply:
hey @harshsinghatz , I've updated the PR with the change requests. let me know if there's anything else I can do to help.